jitpack.yml

```
@@ -0,0 +1,6 @@
+before_install:
+  - wget https://raw.githubusercontent.com/sormuras/bach/releases/11/install-jdk.sh
+  - source install-jdk.sh --feature 17
+  - jshell --version
+install:
+  - ./gradlew clean build publishToMavenLocal
```

I think only `publishToMavenLocal` should suffice: 
```suggestion
  - ./gradlew clean publishToMavenLocal
```

That might avoid running tests for example.